src/v/bytes/tests/iobuf_tests.cc

```diff
@@ -379,38 +379,36 @@ SEASTAR_THREAD_TEST_CASE(iobuf_as_ostream) {
 }
 
 SEASTAR_THREAD_TEST_CASE(alloctor_forward_progress) {
-    static constexpr std::array<uint32_t, 14> src = {{
+    static constexpr auto src = std::to_array<uint32_t>({
       512,
       768,
-      1152,
-      1728,
-      2592,
-      3888,
-      5832,
-      8748,
-      13122,
-      19683,
-      29525,
-      44288,
-      66432,
-      99648,
-    }};
-    static constexpr std::array<uint32_t, 14> expected = {{
+      1280,
+      1792,
+      2560,
+      3584,
+      6144,
+      8192,
+      12288,
+      16384,
+      32768,
+      65536,
+      131072,
+    });
+    static constexpr auto expected = std::to_array<uint32_t>({
       768,
-      1152,
-      1728,
-      2592,
-      3888,
-      5832,
-      8748,
-      13122,
-      19683,
-      29525,
-      44288,
-      66432,
-      99648,
+      1280,
+      1792,
+      2560,
+      3584,
+      6144,
+      8192,
+      12288,
+      16384,
+      32768,
+      65536,
+      131072,
       131072,
```

Comment:
This test just tests that passing size at position N in the table results in size N+1 when calling next alloc size. So the table is copy/paste from the impl, and the second table is the same as the first with the first entry deleted (shifted up by 1).

I also include an additional row in the table testing the case that 128K maps to 128K unchanged.